bundlewrap/items/zfs_dataset.py

```diff
@@ -3,137 +3,154 @@
 from bundlewrap.exceptions import BundleError
 from bundlewrap.items import Item
 from bundlewrap.utils.text import mark_for_translation as _
+from bundlewrap.utils import cached_property
 
 
 class ZFSDataset(Item):
     """
-    Creates ZFS datasets and manages their options.
+    Creates ZFS datasets and manages their properties.
     """
     BUNDLE_ATTRIBUTE_NAME = "zfs_datasets"
     REJECT_UNKNOWN_ATTRIBUTES = False
     ITEM_TYPE_NAME = "zfs_dataset"
+    # for defaults which should be different from 'zfs inherit -S' behaviour
+    PROPERTY_DEFAULTS = {
+        'mountpoint': 'none',
```

Comment:
Don't we want to default to `None` here?

Reply:
None would result in the zfs default value, which is mounting a dataset like `foo/bar/baz` to `/foo/bar/baz`.